README.md

```diff
@@ -42,50 +42,59 @@ extension ImagePipeline {
 Let's start with the basics. Here's an example of how to use a new `RxNuke.Loading` protocol to load an image and display the result on success:
 
 ```swift
-ImagePipeline.shared.loadImage(with: url)
+ImagePipeline.shared.rx.loadImage(with: url)
     .subscribe(onSuccess: { imageView.image = $0.image })
     .disposed(by: disposeBag)
 ```
 
+Or
+
+```swift
+ImagePipeline.shared.rx.loadImage(with: url)
+    .bind(to: imageView.rx.image)
+    .disposed(by: disposeBag)
+```
+
 ### <a name="huc_low_to_high"></a>Going From Low to High Resolution
 
 Suppose you want to show users a high-resolution, slow-to-download image. Rather than let them stare a placeholder for a while, you might want to quickly download a smaller thumbnail first. 
 
 You can implement this using [`concat`](http://reactivex.io/documentation/operators/concat.html) operator which results in a **serial** execution. It would first start a thumbnail request, wait until it finishes, and only then start a request for a high-resolution image.
 
 ```swift
-Observable.concat(pipeline.loadImage(with: lowResUrl).orEmpty,
-                  pipeline.loadImage(with: highResUtl).orEmpty)
-    .subscribe(onNext: { imageView.image = $0.image })
+let lowRes = pipeline.rx.loadImage(with: lowResUrl)
+let highRes = pipeline.rx.loadImage(with: highResUtl)
+
+Observable.concat(lowRes, highRes)
+    .bind(to: imageView.rx.image)
     .disposed(by: disposeBag)
 ```
 
-> `orEmpty` is a custom operator which dismisses errors and completes the sequence instead
-> (equivalent to `func catchErrorJustComplete()` from [RxSwiftExt](https://github.com/RxSwiftCommunity/RxSwiftExt)
-
-
 ### <a name="huc_loading_first_avail"></a>Loading the First Available Image
 
 Suppose you have multiple URLs for the same image. For instance, you might have uploaded an image taken from the camera. In such case, it would be beneficial to first try to get the local URL, and if that fails, try to get the network URL. It would be a shame to download the image that we may have already locally.
 
 This use case is very similar [Going From Low to High Resolution](#huc_low_to_high), but an addition of `.take(1)` guarantees that we stop execution as soon as we receive the first result.
 
 ```swift
-Observable.concat(pipeline.loadImage(with: localUrl).orEmpty,
-                  pipeline.loadImage(with: networkUrl).orEmpty)
+let local = pipeline.rx.loadImage(with: localUrl)
+let network = pipeline.rx.loadImage(with: networkUrl)
+
+Observable.concat(local, network)
```

Comment:
AFAIR `orEmpty` is used here because without it if the first observable (`local`) completes with an error, the resulting observable (created with `concat`) is also going to complete with an error. The `network` observable will never get a chance to execute.